### PR TITLE
#4342 Added image preview div tag

### PR DESCRIPTION
**Pull request by Dhananjay-12**
### Your pull request should…

* [x] have a concise title
* [x] [close an accompanying issue](https://help.github.com/en/articles/closing-issues-using-keywords) if one exists
* [x] [be against the correct branch](https://docs.opencast.org/develop/developer/development-process#acceptance-criteria-for-patches-in-different-versions)
* [x] include migration scripts and documentation, if appropriate
* [x] pass automated tests
* [x] have a clean commit history
* [x] [have proper commit messages (title and body) for all commits](https://medium.com/@steveamaza/e028865e5791)

### CODE REVIEW

File: modules/admin-ui-frontend/app/scripts/shared/partials/modals/event-details.html [hunk 1]
```diff
@@ -1,51 +1,120 @@
-<section ng-show="open" ng-keyup="keyUp($event)" tabindex="1" class="modal modal-animation ng-hide modal-open" id="event-details-modal" ng-controller="EventCtrl">
-
+<section
```
Comment (lkiesow) [inline modules/admin-ui-frontend/app/scripts/shared/partials/modals/event-details.html]: You essentially reformatted the whole file. This makes it more or less impossible to track what actually changed in a given commit. Everyone has his/her own style preferences, but we can't change everything every time we touch a file.

Can you update the patch and change only the part you want to change?

File: modules/admin-ui-frontend/app/scripts/shared/partials/modals/event-details.html [hunk 2]
```diff
@@ -1125,128 +1709,191 @@ <h3>{{ item.name }}</h3>
...
               </tr>
             </table>
           </div>
         </div>
       </div>
+
+      <div
+        class="img-preview"
+        style="background: url('{{ subNavData.url }}') no-repeat;height: 10rem;margin-top: 3%;"
+      ></div>
```
Comment (lkiesow) [inline modules/admin-ui-frontend/app/scripts/shared/partials/modals/event-details.html]: This seems complicated. Doesn't a simple `img` tag work? 
Something like:

```html
<div>
  <img arc="{{ subNavData.url }}" style="max-width: 100%"></img>
</div>
```

**Review by lkiesow — CHANGES_REQUESTED** [review]

**Comment by lkiesow** [comment]
If you have any questions regarding the ICLA, please let me know.
The same goes for the code, of course.

Thanks for taking this on.

File: modules/admin-ui-frontend/app/scripts/shared/partials/modals/event-details.html (hunk 1 above)
Comment (Dhananjay-12) [reply modules/admin-ui-frontend/app/scripts/shared/partials/modals/event-details.html]: I'm really sorry. I forgot disable the prettier extension.

**Pull request closed without merging**